cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -367,10 +367,6 @@ parameter_types! {
 impl pallet_message_queue::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = weights::pallet_message_queue::WeightInfo<Runtime>;
-	#[cfg(feature = "runtime-benchmarks")]
-	type MessageProcessor =
-		pallet_message_queue::mock_helpers::NoopMessageProcessor<AggregateMessageOrigin>;
-	#[cfg(not(feature = "runtime-benchmarks"))]
```

Comment:
@bkontur the problem with the benchmark tests lies here. If the `NoopMessageProcessor` is used, the messages aren't actually processed and so the nonces doesn't increment and all the post-message sending checks fail. It looks like the tests pass with the without the special `NoopMessageProcessor`, will double check CI.